sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/src/Extensions/ActivityExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Diagnostics;
+
+namespace OpenTelemetry.Exporter.AzureMonitor.Extensions
```

Comment:
Do we need this namespace or we could use the `OpenTelemetry.Exporter.AzureMonitor` namespace?

Reply:
I recommended this change.
Raj and I discussed too many classes in the root making it difficult for a new person to learn the project.
I recommended moving a few classes to subdirectories and updating the namespaces to match.

Here specifically, I've always followed this convention of Extension methods belong in unique static classes named "<ObjectName>Extensions" and stored in an "Extensions" namespace and directory.

Reply:
Is this a .NET common practice or not?

Reply:
i've definitely seen this recommended in .NET best practices, but I don't remember where :)

Reply:
I think we should follow https://github.com/dotnet/runtime/blob/6072e4d3a7a2a1493f514cdf4be75a3d56580e84/src/libraries/System.Private.CoreLib/src/System/Reflection/CustomAttributeExtensions.cs

Reply:
> AVOID generic naming of namespaces dedicated to extension methods (e.g., "Extensions"). Use a descriptive name (e.g., "Routing") instead.
> https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/extension-methods

and 

> 3. Put extension methods into their own namespace
> https://devblogs.microsoft.com/vbteam/extension-methods-best-practices-extension-methods-part-6/

The second link is summarizing the first. uncertain where i learned this tip. :(

Reply:
The problem we're trying to solve is that the root directory contains several `internal` classes which are not a part of any critical path. 
Half of these are temporary classes intended to fix the swagger. Most of them are helper classes addressing a specific business case. 
I was trying to address this in a separate PR, but looks like I didn't tag you #14821